blueocean-dashboard/src/main/less/run-details-header.less

```diff
@@ -53,15 +53,16 @@
             white-space: nowrap;
             overflow: hidden;
             text-overflow: ellipsis;
-            
+            align-items: center;
```

Comment:
With the following markup:
```
 branchSlug = (<span className="killSpace">
￼ +                { displayName }
￼ +                <a href={this.pager.branch.branch.url} target="_blank">
￼ +                    <Icon { ...{ style, size: 16, icon: 'launch' } } />
￼ +                </a>
￼ +            </span>);
```

The following css should center that icon. As @cliffmeyers said you need some `flex` in your life ;)
```
+.RunDetailsHeader-sources span.killSpace {
￼ +    display: flex;
￼ +    align-items: center;
￼ +    a svg {
￼ +        margin-left: 8px;
￼ +    }
￼ +}
```